10.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+
+// q10; Find the sum of all the primes below two million.
+
+// function to check is a number is prime; optimsed 
+
+function checkPrime(checkNum){
```

Comment:
Good job!  Its time to learn about the sieve of erasthones. :)

Try to implement this in javascript by following the instructions. 
https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes